Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -1464,7 +1464,7 @@ public class TimeSlider: UIControl {
             let startTimeString = string(for: fullExtentStartTime, style: fullExtentLabelDateStyle)
             fullExtentStartTimeLabel.string = startTimeString
             fullExtentStartTimeLabel.isHidden = !(fullExtentLabelsVisible && isSliderVisible)
-            let startTimeLabelSize: CGSize = startTimeString.size(withAttributes: [kCTFontAttributeName as NSAttributedStringKey: fullExtentLabelFont])
+            let startTimeLabelSize: CGSize = startTimeString.size(withAttributes: [kCTFontAttributeName as NSAttributedString.Key: fullExtentLabelFont])
```

Comment:
I would suggest using `NSAttributedString.Key.font` for all of these:

```suggestion
            let startTimeLabelSize = startTimeString.size(withAttributes: [.font: fullExtentLabelFont])
```